rama-http/src/headers/x_robots_tag/robots_tag_builder.rs

```diff
@@ -0,0 +1,61 @@
+use crate::headers::util::value_string::HeaderValueString;
+use crate::headers::x_robots_tag::custom_rule::CustomRule;
+use crate::headers::x_robots_tag::max_image_preview_setting::MaxImagePreviewSetting;
+use crate::headers::x_robots_tag::robots_tag::RobotsTag;
+
+macro_rules! builder_field {
+    ($field:ident, $type:ty) => {
+        paste::paste! {
+            pub fn [<$field>](mut self, [<$field>]: $type) -> Self {
+                self.0.[<set_ $field>]([<$field>]);
+                self
+            }
+
```

Comment:
same here, you also might want to have the `(mut self, ...) -> Self` variant as an extra